packages/@cosmos-building-blocks/service/src/ecs/ecs-rebalance/index.ts

```diff
@@ -0,0 +1,62 @@
+import { Construct, Duration } from '@aws-cdk/core';
+import { ICluster } from '@aws-cdk/aws-ecs';
+import { Code, Function, Runtime } from '@aws-cdk/aws-lambda';
+import { ManagedPolicy, PolicyDocument, PolicyStatement, Role, ServicePrincipal } from '@aws-cdk/aws-iam';
+import { Rule } from '@aws-cdk/aws-events';
+import { LambdaFunction } from '@aws-cdk/aws-events-targets';
+
+export interface EcsEc2ServiceRebalanceProps {
+  cluster: ICluster;
```

Comment:
Since this is a building block, maybe we can allow `timeout` to be set as a prop...? Eventhough its just an env variable for lambda and consumer can change it from console, its against `everything from code` mantra. What do you think?

Reply:
yep, think that was my intension ... just didnt happen